packs/data/consumable-effects.db/effect-lastwall-soup-greater.json

```diff
@@ -0,0 +1,55 @@
+{
+    "_id": "neZPoQF4hW3A31dd",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @Compendium[pf2e.feats-srd.Gravelands Herbalist]{Gravelands Herbalist}</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Stealth checks against undead</li>\n<li>10 temporary hit points</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>+4 item bonus to Stealth DCs against undead</li>\n</ul>"
```

Comment:
```suggestion
            "value": "<p>Granted by @Compendium[pf2e.equipment-srd.Lastwall Soup (Greater)]{Lastwall Soup (Greater)}</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Stealth checks against undead</li>\n<li>10 temporary hit points</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>+4 item bonus to Stealth DCs against undead</li>\n</ul>"
```

Reply:
Fixed in commit [e58fa08](https://github.com/foundryvtt/pf2e/pull/2596/commits/e58fa08ddcd4be0ff493652b4f8a9085a789fd65)